# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: docshell/base/nsDocShell.cpp
# Commit: f24964c3a231
# Full Hash: f24964c3a231b40d1222d92455c0535a5b820127
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2020-10-22 14:54:09
# Description:
#   Bug 1671697 - Crash in [@ nsDocShell::InternalLoad], r=peterv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93929
# ==============================================================================

diff -r 7d6d66062e84 -r f24964c3a231 docshell/base/nsDocShell.cpp
--- a/docshell/base/nsDocShell.cpp	Wed Oct 21 15:48:23 2020 +0000
+++ b/docshell/base/nsDocShell.cpp	Wed Oct 21 14:22:47 2020 +0000
@@ -9338,7 +9338,7 @@
       nsID historyID = {};
       aLoadState->SHEntry()->GetDocshellID(historyID);
 
-      MOZ_ALWAYS_SUCCEEDS(mBrowsingContext->SetHistoryID(historyID));
+      Unused << mBrowsingContext->SetHistoryID(historyID);
     }
   }
 
